scripts/build.sh

```diff
@@ -12,16 +12,17 @@ cd "${CONSTRUCT_ROOT}"
 # Constructor should be latest for non-native building
 # See https://github.com/conda/constructor
 echo "***** Install constructor *****"
-conda install -y "constructor>=3.3.1" jinja2 curl libarchive -c conda-forge --override-channels
-
+conda install --yes \
+    --channel conda-forge --override-channels \
+    jinja2 curl libarchive \
+    "constructor>=3.3.1=*_1"
```

Comment:
This means `version>=3.3.1` and `build=*_1`. Is that what you mean?

Reply:
yeah. do i need `==` in the second one?

Reply:
Actually, i mean "3.3.2" and "build=0" would be ok.

I'm not sure if syntax exists for that.

Reply:
Yeah, there's no syntax for that. Let's keep it `constructor>=3.3.1` and hope for the best.

Reply:
ok, i hope the smaller test matrix passes now for the ppc64le. it has been quite slow.